app/Mail/QuizResults.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Mail;
+
+use App\Models\Quiz;
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Mail\Mailables\Content;
+use Illuminate\Mail\Mailables\Envelope;
+use Illuminate\Queue\SerializesModels;
+
+class QuizResults extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    public Quiz $quiz;
+
+    public string $name;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct(Quiz $quiz, string $name)
+    {
+        $this->quiz = $quiz;
+        $this->name = $name;
+    }
+
+    /**
+     * Get the message envelope.
+     *
+     * @return \Illuminate\Mail\Mailables\Envelope
+     */
+    public function envelope()
+    {
+        return new Envelope(
+            subject: __('Quiz Results for ').$this->name,
```

Comment:
Probably better to write this as 

```suggestion
    {
        return new Envelope(
            subject: __('Quiz Results for :name ', ['name' => $this-name]),
```

In doing so you allow for more flexibility in the localization. That is, the localized version may want to have the name appear in a different part of the string or perhaps with some punctuations or additional content after the name.

Reply:
You should review the string concatenations through out. Anything that is sent for translation should interpolate as the suggestion above.